bin/run.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```

Comment:
Add a comment here, and in the README (or link the one to the other) so we know what the purpose of this file is.

Reply:
Ok, adding comment